src/@commonsku/styles/Artwork.tsx

```diff
@@ -105,7 +125,11 @@ export const Artwork = (props: {picture?:string, icon?:string, name:string, cssH
     <ArtworkInfo withPicture={props.picture?true:false} >
       {props.edit && props.onSave ?
         <div style={{display:"flex"}}>
-         <Input style={{flexGrow:1, marginBottom: 0, fontSize: ".8rem", padding: ".3rem"}} value={props.name}/>
+         {props.inputEl || <Input // Custom or default Input Element
```

Comment:
What's the use case for passing a custom input element? If it's something other than styling, maybe we should also apply default styles to it so that you don't have to write those within the application?